codemp/rd-rend2/tr_backend.cpp

```diff
@@ -2029,6 +2029,19 @@ static void RB_RenderSSAO()
 	GL_BindToTMU(tr.hdrDepthImage, TB_COLORMAP);
```

Comment:
There are two texture samplers in the SSAO shader, but this is the only texture that gets bound. There should be another bind for the second texture...

Reply:
I thought it was already bound. Where can I find TB_LIGHTMAP?